grpc-mqtt.cabal

```diff
@@ -9,7 +9,10 @@ author:      Arista Networks
 maintainer:  opensource@awakesecurity.com
 copyright:   2021-2022 Arista Networks
 category:    Network
-build-type:  Simple
+build-type:  Custom
+
+custom-setup
+  setup-depends: base, Cabal, filepath
 
 extra-source-files:
```

Comment:
These extra source files are no longer correct, and I believe they are no longer needed since the `.proto` files fall under the `hs-source-dirs`.

Reply:
Removed by 89c11c99d9f5bc62dce2fdcb542740ed5f8ece1b .  Thanks.